### ISSUE TITLE
parse special symbols properly

### ISSUE THREAD

**Issue by renanbr** [renanbr/bibtex-parser#5]
Discussion started at https://github.com/raphael-st/bibtex-parser/commit/f2a9aa9c6d7506a26a3701dbb70dd879b4c78e5a

Reference: http://www.bibtex.org/SpecialSymbols/

**Comment by renanbr** [comment]
The commits https://github.com/renanbr/bibtex-parser/commit/dfa64f7d0136d377f4ec4558d547a048f5c9dac3 and https://github.com/renanbr/bibtex-parser/commit/554e41b62190f931885b25db9800d11db3a1b61b provide a way to process tag values.

I'm still looking for some library that can translate latex special symbols to utf-8.

**Comment by renanbr** [comment]
@raphael-st, I've found wrappers for the [pandoc](http://pandoc.org/) command: [ryakad/pandoc-php](https://github.com/ryakad/pandoc-php) and [ondrs/pandoc-php](https://github.com/ondrs/pandoc-php)

Using one of these libraries in conjunction with `setTagValueProcessor()`, [available since v0.3.0](https://github.com/renanbr/bibtex-parser/releases/tag/0.3.0), you can translate latex into what you want

~~~php
$pandoc = new Pandoc\Pandoc();
$listener->setTagValueProcessor(function (&$value, $tag) use ($pandoc) {
    $value = $pandoc->runWith($value, [
        "from" => "latex",
        "to" => "plain" // or "html"
    ]);
});
~~~
